frontend/web/components/Aside.js

```diff
@@ -91,8 +91,9 @@ const Aside = class extends Component {
 
   render() {
     const { asideIsVisible, disabled, toggleAside } = this.props
-    let integrations = Utils.getFlagsmithValue('integrations') || '[]'
-    integrations = JSON.parse(integrations)
+    const integrations = Object.keys(
+      JSON.parse(Utils.getFlagsmithValue('integration_data') || '{}'),
+    )
```

Comment:
I'd argue it's confusing that we now have `integrations` defined in separate places, using the same Flagsmith flag, but holding a different representation of the data. I'd recommend renaming one of the variables, but I'm not going to die on that hill so I'll approve regardless.